ToyBox/classes/MainUI/BagOfTricks.cs

```diff
@@ -554,7 +554,9 @@ public static class BagOfTricks {
                 () => Toggle("Free Meta-Magic".localize(), ref Settings.toggleMetamagicIsFree),
                 () => Toggle("No Fog Of War".localize(), ref Settings.toggleNoFogOfWar),
                 () => Toggle("Restore Spells & Skills After Combat".localize(), ref Settings.toggleRestoreSpellsAbilitiesAfterCombat),
-                () => UI.Toggle("Recharge Items After Combat", ref settings.toggleRechargeItemsAfterCombat),
+                () => Toggle("Restore Just Spells After Combat".localize(), ref Settings.toggleRestoreSpellAfterCombat),
+                () => Toggle("Restore Just Abilities After Combat".localize(), ref Settings.toggleRestoreAbilitiesAfterCombat),
+                () => Toggle("Recharge Items After Combat", ref settings.toggleRechargeItemsAfterCombat),
```

Comment:
.localize() something something (no need to add the string; those get automatically generated when I update the localization files next time. Having .localize there would be good though

Reply:
Forgot that the other day